### PR TITLE
Feature/define polices#3

### LINKED ISSUE #3: Define Roles
use Cancan or  Rollify

### PR DESCRIPTION
**Pull request by edman193**
closes #3 

Roles with rolify and cancan

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -16,4 +19,12 @@ def self.from_omniauth(auth)
       user.image = auth.info.image # assuming the user model has an image
     end
   end
+
+  def assign_role
+    add_role(:user)
+  end
+
+  def assign_role
```
Comment (jclopezdev) [inline app/models/user.rb]: This method is duplicated

Comment (edman193) [reply app/models/user.rb]: Yes, I don't know what happend

**Comment by jclopezdev** [comment]
But you have "spec/factories/roles.rb" in plural could you change user for users in that file? @edman193

**Comment by edman193** [comment]
yes, I'm going to change it

**Comment by edman193** [comment]
I already change the file's name

**Comment by jnegrete** [comment]
@edman193, I think that is better modify the field instead of delete it, I don't know if it was generated automatically or written by @jclopezdev, only remember to keep the history.

**Comment by edman193** [comment]
ok, I'll keep in mind

**Pull request merged**